tests/e2e/utils/customer.js

```diff
@@ -100,9 +100,12 @@ export async function checkout( page ) {
 		await page.locator( '#billing-state input' ).fill( user.statename );
 	}
 
+	//TODO: See if there's an alternative method to click the button without relying on waitForTimeout.
+	await page.waitForTimeout( 3000 );
```

Comment:
I commented out this line and ran the related tests several times, and all attempts can pass. 

> * I added `await page.waitForTimeout( 3000 );` because it was clicking the button too quickly making the tests fail.

Maybe an alternative method is to wait for the button to be enabled:

```suggestion
	await expect( page.locator( 'text=Place order' ) ).toBeEnabled();
```

Reply:
Thanks for the suggestion! Locally the E2E tests failed for me, even with the suggestion that you mentioned. For now, I will keep it as it is. If I find the root cause I will change it.